assets/js/modules/analytics-4/components/widgets/PopularAuthorsWidget.stories.js

```diff
@@ -79,6 +80,7 @@ Ready.args = {
 	},
 };
 Ready.scenario = {
+	// eslint-disable-next-line sitekit/no-storybook-scenario-label
```

Comment:
@benbowler the occurrence of these throughout the PR is because if we remove the label now, then there are a bunch of VRT image updates to make that are completely irrelevant to this body of work. We can (if need be) create a follow up or catch-all issue to remove scenario labels across the entire codebase.

Reply:
Ah yes, I checked and there is an issue for this #9103